web/static/js/prom_console.js

```diff
@@ -502,7 +502,7 @@ PromConsole.Graph.prototype.dispatch = function() {
   var pending_requests = this.params.expr.length;
   for (var i = 0; i < this.params.expr.length; ++i) {
     var endTime = this.params.endTime;
-    var url = "/api/query_range?expr=" + encodeURIComponent(this.params.expr[i])
+    var url = PATH_PREFIX + "api/query_range?expr=" + encodeURIComponent(this.params.expr[i])
```

Comment:
Where are you getting PATH_PREFIX from here?

I think you need to add it in prom.lib